src/textual_dev/tools/diagnose.py

```diff
@@ -93,7 +93,17 @@ def _guess_term() -> str:
         if term_version is not None:
             term_program = f"{term_program} ({term_version})"
 
-    return "*Unknown*" if term_program is None else term_program
+    # Seems we can't work this out.
```

Comment:
I see these changes in this commit but I don’t see them in the final “changes” of the PR. What am I missing?

Reply:
The fact that I was working on a fork that I forgot to sync, was confused as to why the `screen` detection wasn't in the code, added it to this, then realised what was going on, did a sync, a merge, and then it had the effect of nullifying that change.